### PR TITLE
Multiple borrowers

### PR DESCRIPTION
**Pull request by RuvendeGroote**

### CODE REVIEW

File: src/react/components/DealOverview.tsx [hunk 1]
```diff
@@ -99,16 +101,37 @@ export const DealOverview = () => {
...
+	useEffect(() => {
 		if (wallet?.publicKey && connection.connection) {
 			fetchDealData();
 		}
-	}, [connection.connection, wallet?.publicKey, fetchDealData]);
+	}, [wallet, connection.connection, fetchDealData]);
+
+	useEffect(() => {
+		if (wallet?.publicKey && connection.connection) {
+			setPlaceholder("0");
```
Comment (BouweCeunen) [inline src/react/components/DealOverview.tsx:139]: maybe another placeholder for the key? `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx` or something?

Comment (RuvendeGroote) [reply src/react/components/DealOverview.tsx:139]: there is a ticket for this

Comment (RuvendeGroote) [reply src/react/components/DealOverview.tsx:139]: we discussed it this morning during the call and a ticket was made

Comment (BouweCeunen) [reply src/react/components/DealOverview.tsx:139]: indeed, didn't know there was already a ticket 👍

**Review by SushantChandla — DISMISSED** [review]

**Review by BouweCeunen — DISMISSED** [review]

**Review by BouweCeunen — APPROVED** [review]

**Review by SushantChandla — APPROVED** [review]

File: src/react/components/DealsTable.tsx [hunk 2]
```diff
@@ -45,40 +55,47 @@ export const DealsTable = () => {
...
 				<TableCell>{createdAt.toUTCString()}</TableCell>
-				<TableCell>{toUIPercentage(deal.financingFeePercentage)}%</TableCell>
 				<TableCell>{(goLiveAt && goLiveAt.toUTCString()) || "-"}</TableCell>
+				<TableCell>{toUIPercentage(deal.financingFeePercentage)}%</TableCell>
 				<TableCell>{millify(toUIAmount(deal.principal.toNumber()))}</TableCell>
 				<TableCell>{millify(toUIAmount(deal.principalAmountRepaid.toNumber()))}</TableCell>
 				<TableCell>{millify(toUIAmount(deal.interestAmountRepaid.toNumber()))}</TableCell>
 				<TableCell>{`${daysRemaining} / ${deal.timeToMaturityDays}`}</TableCell>
 				<TableCell>{`${dealStatus !== null && formatDealStatus(dealStatus)}`}</TableCell>
+				<TableCell>{userDeal && "repay"}</TableCell>
```
Comment (SushantChandla) [inline src/react/components/DealsTable.tsx:77]: We can improve styling here : )

**Pull request merged**